src/Fontconfig.jl

```diff
@@ -1,25 +1,17 @@
 module Fontconfig
 
-depsfile = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
-if isfile(depsfile)
-    include(depsfile)
-else
-    error("Fontconfig.jl not properly installed. Please run `Pkg.build(\"Fontconfig\")` " *
-          "and restart Julia.")
-end
-
-import Base.Sys
 using Printf
+using Fontconfig_jll
 
 export format, match, list
 
 function __init__()
-    ENV["FONTCONFIG_FILE"] = joinpath(dirname(jl_libfontconfig), "..", "etc", "fonts", "fonts.conf")
-    ccall((:FcInit, jl_libfontconfig), UInt8, ())
+    ENV["FONTCONFIG_FILE"] = joinpath(dirname(libfontconfig), "..", "etc", "fonts", "fonts.conf")
```

Comment:
`fonts_conf` is [one of the products](https://github.com/JuliaPackaging/Yggdrasil/blob/930b004d197192c527c259f52ef4475ae2f848b4/F/Fontconfig/build_tarballs.jl#L68) exactly to make this easier:
```suggestion
    ENV["FONTCONFIG_FILE"] = fonts_conf
```
:wink: